### DISCUSSION TITLE
Conflito de dependência com Toaster

### DISCUSSION THREAD

**Discussion by GabrielWerlich** [balta-io/7187#80] [Perguntas e Respostas] [answered]
Balta eu sei que por regra o ideal seria fazer o curso criando o projeto do angular na mesma versão do curso, porém estou quase completando o curso na ultima versão (13). Na aula 27 "Toastr" quando eu tento instalar os pacotes acontece o seguinte:

1 - npm install ngx-toastr --save  (Instala com sucesso)
2 - npm install @angular/animations --save (Acontece o erro abaixo)

"Could not resolve dependency: npm ERR! @angular/animations@"~13.2.0" from the root project

É um conflito de versões pelo o que entendi, mas porque isso acontece? O projeto do Toaster está desatualizado em relação ao Angular 13? 

![image](https://user-images.githubusercontent.com/40069738/159145556-d1af364f-98af-497b-82e7-60eb9372a381.png)

![image](https://user-images.githubusercontent.com/40069738/159145547-33c74b37-e652-40a3-9f80-6237d275372b.png)

**Comment by andrebaltieri** [comment, marked as answer]
@GabrielWerlich Imagino que sim, faz tempo que não vejo o pessoal falar sobre o Toastr, infelizmente.
Neste caso, tem que dar uma olhada no site deles e ver se tem suporte (Se sim, como faz) para a versão 13 do Angular.
